apps/mull-ui/src/app/pages/event-page/event-page-header/event-page-header.scss

```
@@ -1,4 +1,6 @@
-.event-page-header {
+@import '../../../../variables.scss';
+
+.event-page-header-container {
   padding: 0.5rem 0.5rem 0;
   box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.1);
 }
```

Adding the z-index will fix the problem of the event-info spilling up

```suggestion
.event-page-header-container {
  padding: 0.5rem 0.5rem 0;
  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.1);
  z-index: 1;
}
```

---

For some reason, I removed that when experimenting and forgot to put it back, my bad. As for the header content spilling over, I'm working on a fix for that.